frontend/lib/identification/info_dialog.dart

```diff
@@ -26,7 +26,9 @@ class InfoDialog extends StatelessWidget {
         title: Text(title, style: theme.textTheme.headlineSmall),
       ),
       content: child,
-      actions: [TextButton(onPressed: () => Navigator.of(context).pop(), child: Text(t.common.ok))],
+      actions: [
+        TextButton(onPressed: () => Navigator.of(context, rootNavigator: true).pop(), child: Text(t.common.ok))
+      ], //TODO make close an prop
```

Comment:
what does that mean?

Reply:
will delete was just a remindet for myself